cpp/benchmarks/common/generate_input.hpp

```diff
@@ -403,3 +403,35 @@ std::unique_ptr<cudf::table> create_random_table(std::vector<cudf::type_id> cons
                                                  row_count num_rows,
                                                  data_profile const& data_params = data_profile{},
                                                  unsigned seed                   = 1);
+
+/**
+ * @brief Generate sequence columns starting with value 0 in first row and increasing by 1 in
+ * subsequent rows.
+ *
+ * If the number of passed types is smaller than the number of requested column, the columns types
+ * with be repeated in round-robin order to fill the table.
```

Comment:
I think "repeated" (not round robin) is the right word here. Round-robin is how you would schedule a tournament so that all teams play one another pair-wise. https://en.wikipedia.org/wiki/Round-robin_tournament

```suggestion
 * If the number of passed types is smaller than the number of requested column, the types
 * will be repeated to fill the number of columns requested.
```